server/routes.go

```diff
@@ -308,6 +308,14 @@ func (s *Server) EmbedHandler(c *gin.Context) {
 		truncate = false
 	}
 
+	r, m, opts, err := s.scheduleRunner(c.Request.Context(), req.Model, []Capability{}, req.Options, req.KeepAlive)
```

Comment:
We may have moved this up too high. This will cause the model to load on invalid input